lib/perl/Genome/Model/Tools/Annotate/AddRsid.pm

```diff
@@ -79,39 +92,70 @@ sub execute {
             my @var_alleles = @tmp1;
             @var_alleles = @tmp2 if (scalar(@tmp2) > scalar(@tmp1));
 
-            # Gets the list of allele frequencies if CAF nomenclature is used
-            if($INFO =~ /(CAF=[0-9.,]+)/) {
-                $caf_string = ($INFO =~ /(CAF=[0-9.,]+)/)[0];
-                @af = split(/,/, $caf_string);
-                # The first allele frequency provided is the reference allele which we don't want so drop it
-                shift @af;
+            # Script not designed to handle indels, ignores them if present
+            my $is_indel = 0;
+            for (@var_alleles) {
+                $is_indel = 1 if length($_) > 1;
+            }
+            if (length($ref) > 1 || $is_indel == 1) {
+                next;
             }
 
             my @dbSNPids = split_dbSNPBuildID($INFO);
 
-            for (my $i = 0; $i < @var_alleles; $i++) {
-                # Reassign GMAF value to the CAF value, if one was found. Pull the appropriate var allele
-                if(scalar(@af) > 0){
-                    $GMAF = $af[$i];
-                    unless($GMAF){
-                        $self->warning_message("CAF undefined for allele, i: $i, CAF values undefined: $caf_string, non-reference AFs: @af");
-                    }
-                    if($GMAF eq '.'){
-                        $GMAF = "-";
-                    }   
+            if($file_format eq "CAF") {
+                # Gets the list of allele frequencies if CAF nomenclature is used
+                if($INFO =~ /(CAF=[0-9.,]+)/) {
+                    $caf_string = ($INFO =~ /(CAF=[0-9.,]+)/)[0];
```

Comment:
Crap, except I forgot to implement these changes before merging the pull request.